content/en/docs/Writing policies/generate-validating-admission-policies.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Generate ValidatingAdmissionPolicies
+description: >
+  Generate Kubernetes ValidatingAdmissionPolicies based on `validate.cel` subrules.
+weight: 50
+---
+While Kubernetes [ValidatingAdmissionPolicies](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/) provides a declarative, in-process option for validating admission webhooks and uses the [Common Expression Language](https://github.com/google/cel-spec) (CEL) to perform resource validation checks directly in the API server, it falls short of the features that can be provided by Kyverno policies. Kyverno policies are capable of performing complex validation, mutation, generation, image verification, reporting, and off-cluster validation, whereas ValidatingAdmissionPolicies cannot. Since it is important to unify the policy management used in clusters, Kyverno policies can be used to generate Kubernetes ValidatingAdmissionPolicies. This feature allows the process of resource validation to take place in the API server instead.
```

Comment:
Maybe a new paragraph here stating that this capability is in alpha state, disabled by default (opt-in only), and comes with many restrictions as to what types of Kyverno validate rules can result in VAP generation.

Reply:
Also state in this opening paragraph that VAP is not fully enabled by default as of Kubernetes versions up to and including 1.28. State the minimum version of Kubernetes where this will work as well.